<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagerDialog</class>
 <widget class="QDialog" name="UserManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
font-size:14px;
}
QPushButton {
	background-color: rgb(173, 216, 230);
	border:2px groovegray;
	border-radius:10px;
	padding:2px 4px;
}

QPushButton:hover {
/*   background-color: khaki;	*/
	background-color: rgb(135, 206, 250);
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
	background-color: rgb(70, 130, 180);
/*    background-color: #d0d67c;	*/
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="userManagerHLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="userManager" native="true">
       <layout class="QVBoxLayout" name="userVLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="userHLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="userInfoTableview"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="operateWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <widget class="QPushButton" name="addUserBtn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>添加用户</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifyBtn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>修改密码</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delUserBtn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>删除用户</string>
        </property>
       </widget>
       <widget class="QPushButton" name="closeBtn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;宋体&quot;;</string>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
